### PR TITLE
GH-860: Doc updates

### PR DESCRIPTION
**Pull request by plabadie**
closes GH-860

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ Spartacus provides core storefront features such as:
 # Requirements
 
 - SAP Commerce Cloud instance (Release 1808)
```
Comment (LTiger14) [inline README.md]: Should the version be 1811? Or both 1808 and 1811?

File: README.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ Spartacus provides core storefront features such as:
...
-- Angular CLI (https://angular.io/): >= 6.2.4 and < 7.0.0
-- node.js >= 8.9.0
-- yarn >= 1.6.0
+- Angular CLI (https://angular.io/): >= v7.2.1 and < 8.0.0
```
Comment (LTiger14) [inline README.md]: It's small bu why does 7.2.1 get a `v` 😉 ?

File: README.md (hunk 1 above)
Comment (plabadie) [reply README.md]: Good catch.  As far as I know, for now, we support the latest version of SAP Commerce.

File: README.md (hunk 2 above)
Comment (plabadie) [reply README.md]: The more consistent the better.

7.2.1 isn't that special.  I removed the v 🙂.

**Pull request merged**